package-build/package-build.el

```diff
@@ -1030,6 +1030,24 @@ line per entry."
     (setq entries (sort entries (lambda (a b)
                                   (string< (symbol-name (car a))
                                            (symbol-name (car b))))))
+    (when (>= emacs-major-version 29)
```

Comment:
No, but it's only relevant when run under Emacs 29 so I thought it might be useful for people running their own melpa's (if any)

Happy to remove though